apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -120,6 +134,20 @@ export class CreateIntegration {
         });
       }
 
+      if (command.channel === ChannelTypeEnum.EMAIL && command.providerId !== EmailProviderIdEnum.Novu) {
```

Comment:
we need to deactivate npvu provider if they create another type of integration

Reply:
Besides covering the case that @ainouzgali raised that is very important, what do you think about creating an use case to isolate this logic, temptively: `DisableNovuIntegration` where we could you pass in the command the `channel` and the `environmentId`? Based on the channel we would know which `providerId` to use.